### ISSUE TITLE
Update LDAP TLS certificate

### ISSUE THREAD

**Issue by penguian** [ACCESS-NRI/accessdev-Trac-archive#186]
`keyword_TIWG` `resolution_fixed`   |    by saw562@nci.org.au

___

Via Chris: NCI's LDAP certificate expires 2015-06-05

___
_Issue migrated from [trac:186](https://accessdev.nci.org.au/trac/186) at 2024-01-31 18:12:39 +1100_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
**Branch**: [/log/puppet?rev=saw562/ldap-cert saw562/ldap-cert]

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
No issues with certificate on dev-test, will apply to production

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _changed status from `new` to `closed`_

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _changed owner from `` to `Scott Wales <scott.wales@unimelb.edu.au>`_

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _changed resolution from `` to `fixed`_

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _commented_


___
In [changeset:"a688bc1d578d56758281cf48e45aed0c9018c5b5/puppet"]:
```
#CommitTicketReference repository="puppet" revision="a688bc1d578d56758281cf48e45aed0c9018c5b5"
Fixes #186 Update LDAP TLS cert

Merge branch 'ldap'
```
